integration-cli/docker_cli_exec_test.go

```diff
@@ -560,6 +561,10 @@ func (s *DockerSuite) TestExecWindowsOpenHandles(c *check.C) {
 			// The initial exec process (cmd.exe) has exited, and both sleeps are currently running
 			break
 		}
+		count++
+		if count >= 30 {
+			c.Error("too many retries")
```

Comment:
Total forgot about these. Well I updated to c.Fatal in this test - that's consistent with -most- of the rest of the CI suite, and I know it doesn't cause the entire suite to abort. Whether the other `.Error` calls littered in various files should be updated, I'll leave for a future PR and discussion....